sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
```

Comment:
Sorry if I just noticed this, but *eutils* doesn't seem to be used by this ebuild at all.

Reply:
Thanks, dropped